Client/Frontend/UserContent/UserScripts/MainFrame/AtDocumentEnd/MetadataHelper.js

```diff
@@ -21,8 +21,13 @@ function MetadataWrapper() {
     if (location.pathname.startsWith("/amp/")) {
       let canonicalLink = document.querySelector("link[rel=\"canonical\"]");
```

Comment:
`suggestion:` We can use `'` instead of escaping `"` 
```suggestion
      const canonicalLink = document.querySelector("link[rel='canonical']");
```